docs/5-call-convenience-methods.md

```diff
@@ -0,0 +1,117 @@
+## Getting Started with the Nexmo Conversation iOS SDK
+
+In this getting started guide we'll cover adding call methods to the Conversation we created in the [simple conversation with audio](/stitch/in-app-messaging/guides/4-enable-audio/swift) getting started guide. We'll deal with member call events that trigger on the application and call state events that trigger on the Call object.
+
+The main difference between using these Call convenience methods and enabling and disabling the audio in the previous quickstart is that these methods do a lot of the heavy lifting for you. By calling a user directly, a new conversation is created, and users are automatically invited to the new conversation with audio enabled. This can make it easier to start a separate direct call to a user or start a private group call among users.
+
+## Concepts
+
+This guide will introduce you to the following concepts.
+
+- **Calls** - calling an User in your application without creating a Conversation first
+- **Call Events** - CallEvent event that fires on an `ConversationClient` or `Call`
+
+
+### Before you begin
+- Ensure you have run through the [previous guide](/stitch/in-app-messaging/guides/4-enable-audio/swift) 
+
+### 1.0 - Updating iOS App
+
+We will use the application we already created for [the first audio getting started guide](/stitch/in-app-messaging/guides/4-enable-audio/swift). All the basic setup has been done in the previous guides and should be in place. We can now focus on updating the client-side application.
+
+### 1.1 Adding a CallViewController with `.storyboard` files 
+
+### 1.2 Initiating a call 
+
+Pass a list of users to initiate a call object:
+
+```swift
+do {
+    let users = ["user1", "user2"]
+
+    try client.media.call(users, onSuccess: { result in
+        // code here
+        // result contains Call object and any errors from requesting invites for users
+    }, onError: { networkError in
+        // code here
+    })
+} catch let error {
+    // code here
```

Comment:
`//code here` isn't helpful to developers. We should spell out better what kind of code goes here or what you could do. Something like `//show error in UI`